docs/usage.md

```diff
@@ -87,7 +87,7 @@ const space = await spaces.get('demoSlideshow', { offlineTimeout: 180_000, curso
 
 ## Members
 
-Members is a core concept of the library. When you enter a space, you become a `member`. On the client, your own membership is to referred to as `self`. You can get your `self` by calling `space.getSelf`. To get all the members (including self), call `space.getMembers`. These method will return (respectively an object and array of):
```

Comment:
Similar to other files, we need some more information on the `space` object before moving to `space.members`